core/templates/dev/head/services/ExtensionTagAssemblerService.ts

```diff
@@ -17,22 +17,30 @@
  * the learner and editor views.
  */
 
-require('filters/string-utility-filters/camel-case-to-hyphens.filter.ts');
-require('services/HtmlEscaperService.ts');
+import { downgradeInjectable } from '@angular/upgrade/static';
```

Comment:
Aren't you missing the injectable decorator?

Reply:
Weird :/ will fix that @bansalnitish